apps/server/src/shared/common/interceptor/timeout.interceptor.spec.ts

```diff
@@ -20,8 +20,7 @@ describe('TimeoutInterceptor', () => {
 		});
 
 		it('should respond with error code if request runs into timeout', async () => {
```

Comment:
Is there a way to mock a route and attach the decorator, so we can test the or condition `timeoutValue || this.requestTimeout` of the interceptor?

Reply:
see: 
apps/server/src/shared/common/interceptor/test/create-test.module.ts